OrbitQt/servicedeploymanager.h

```diff
@@ -69,6 +69,8 @@ class ServiceDeployManager : public QObject {
       OrbitSshQt::SftpOperation::FileMode dest_mode);
   outcome::result<void> StopSftpChannel(OrbitSshQt::SftpChannel*);
 
+  void StartWatchdog(std::string_view start_passphrase = "start_watchdog",
```

Comment:
Nit: Could you put the text into a kWtachdogStartPassphrase variable into the implementation file?

Reply:
done